`_strtof_l` isn't included in the MinGW headers. Use `_strtod_l` instead
and convert to float.
diff -urN opencolorio-2.1.2/src/utils/NumberUtils.h.orig opencolorio-2.1.2/src/utils/NumberUtils.h
--- opencolorio-2.1.2/src/utils/NumberUtils.h.orig	2022-07-15 20:33:05.367178000 +0200
+++ opencolorio-2.1.2/src/utils/NumberUtils.h	2022-07-15 20:33:21.423287000 +0200
@@ -98,8 +98,10 @@
     char *endptr = nullptr;
 
     float
-#ifdef _WIN32
+#ifdef _MSC_VER
     tempval = _strtof_l(first, &endptr, loc.local);
+#elif defined (__MINGW32__)
+    tempval = _strtod_l(first, &endptr, loc.local);
 #elif __APPLE__
     // On OSX, strtod_l is for some reason drastically faster than strtof_l.
     tempval = static_cast<float>(::strtod_l(first, &endptr, loc.local));
